nextflow_schema.json

```diff
@@ -52,6 +52,10 @@
                     "type": "string",
                     "description": "MultiQC report title. Printed as page header, used for filename if not otherwise specified.",
                     "fa_icon": "fas fa-file-signature"
+                },
+                "nf_core_pipeline": {
+                    "type": "string",
+                    "description": "Create a samplesheet for the specified nf-core pipeline"
```

Comment:
Missing longer helptext describing a bit more what it does; also missing the `enum` to ensure the dropdown/validation is correct.